src/player.cpp

```diff
@@ -487,36 +527,14 @@ void GameController::handleAnalog(int player)
 	}
 }
 
-int GameController::getLeftXMove() // with sensitivity
-{
-	if (!isActive())
-	{
-		return 0;
-	}
-	int x = getRawLeftXMove();
-	x *= gamepad_leftx_sensitivity;
-	return x;
-}
-
-int GameController::getLeftYMove() // with sensitivity
-{
-	if (!isActive())
-	{
-		return 0;
-	}
-	int y = -getRawLeftYMove();
-	y *= gamepad_lefty_sensitivity;
-	return y;
-}
-
 int GameController::getRightXMove() // with sensitivity
 {
 	if (!isActive())
 	{
 		return 0;
 	}
 	int x = getRawRightXMove();
-	x *= gamepad_rightx_sensitivity;
+	x *= getGamepadRightXSensitivity(0);
```

Comment:
should this not take a proper player index?

Reply:
Done - splitscreen will need this sensitivity unique for each player since everyone just uses the global sensitivity